### DISCUSSION TITLE
Possible to "Set as default" automatically upon accessing DB?

### DISCUSSION THREAD

**Discussion by Robin-Britz** [dbeaver/dbeaver#20771] [Q&A]
I've got a server that has 20+ DB's on it (MS SQL).
Is there a way to have "Set as default" automatically (when right clicking database) to be set on the DB as soon as I drop it down and navigate to stored procedures?

I work on all these DB's throughout the day and it's quite tedious to click the option "Set as default" or use CTRL + SHIFT + A every time I switch DB's to update or amend stored procedures.

**Comment by LonwoLonwo** [comment]
Hello @Robin-Britz 

You can set the default database in your connection setting for an existing connection, as an option.

![2023-08-22 10_59_42-Window](https://github.com/dbeaver/dbeaver/assets/45152336/2c9a0c54-abd1-4428-ac77-c92d9f35fac4)

**Reply by Robin-Britz** [reply]
Thank you for your response - this definitely does work, but is not an optimal solution unfortunately - as my 1 server has 20+ databases. Setting the default database to one of the 20+ databases before starting to work on said DB - editing Stored Procedures for instance - is quite tedious.
